sakura_core/dlg/CDialog.h

```diff
@@ -93,7 +91,7 @@ class CDialog{
 	virtual BOOL OnSize( WPARAM wParam, LPARAM lParam );
 	virtual BOOL OnMove( WPARAM wParam, LPARAM lParam );
 	virtual BOOL OnDrawItem( WPARAM wParam, LPARAM lParam ){return TRUE;}
-	virtual BOOL OnTimer( WPARAM wParam ){return TRUE;}
+	virtual BOOL OnTimer( HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam ){return TRUE;}
```

Comment:
プロトタイプは windowsx.h の定義に合わせたらいいと思います。
[WM_TIMERメッセージ](https://learn.microsoft.com/ja-jp/windows/win32/winmsg/wm-timer)の戻り値は使われないので、戻り値型は void で構いません。

```suggestion
	virtual void    OnTimer(HWND hWnd, UINT id) { /* do nothing */ }
```